crystal/airootfs/etc/packages.x86_64

```diff
@@ -29,19 +32,14 @@ mesa
 firefox
 
 # Things we're hosting
-arch-install-scripts
-base
-filesystem
 neofetch
-pfetch
-grub
 
 # Our in-house
-jade-gui
+crystal-core
+crystal-branding
 jade
+jade-gui
 onyx
-ame
```

Comment:
why was amethyst removed?

Reply:
Because it is now a part of the `crystal-core` metapkg (https://github.com/crystal-linux-packages/crystal-core/blob/main/PKGBUILD)